src/pybind/mgr/cephadm/module.py

```diff
@@ -921,7 +783,7 @@ def set_ssh_user(self, user):
         self._reconfig_ssh()
 
         host = self.cache.get_hosts()[0]
-        r = self._check_host(host)
+        r = CephadmServe(self)._check_host(host)
```

Comment:
```suggestion
        r = self.serve._check_host(host)
```

Reply:
doesn't make a difference in the current impl., but it we were to add any additional state to the `CephadmServe` class, this instance might not match the one created during `serve()`

Reply:
Actually it was my intention to call things within `CephadmServe` very awkward. IMO only `serve()` should create an instance and IMO no one else should have the need to use it.